install_config/configuring_aws.adoc

```diff
@@ -218,3 +220,61 @@ $ systemctl restart origin-node
 ----
 endif::[]
 .  Add back any xref:../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[labels on each node] that you previously had.
+
+[[aws-cluster-labeling]]
+== Labeling Clusters for AWS
+Starting with {product-title} version 3.7 of the installation program, if you
```

Comment:
What does "for the installation program" refer to? I don't think this is an OpenShift thing. Would it be easier to take it out?